src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -165,7 +165,7 @@ CHIP_ERROR OperationalCredentialsAttrAccess::ReadFabricsList(EndpointId endpoint
 
             fabricDescriptor.fabricIndex = fabricInfo.GetFabricIndex();
             fabricDescriptor.nodeId      = fabricInfo.GetPeerId().GetNodeId();
-            fabricDescriptor.vendorId    = fabricInfo.GetVendorId();
+            fabricDescriptor.vendorId    = static_cast<chip::VendorId>(fabricInfo.GetVendorId());
```

Comment:
Should we fix FabricInfo to return the right thing?

Reply:
I agree that it would be the right thing to do. The thing is that `uint16_t` instead of `chip::VendorId` is used in many places in the code.
At the end of the day someone motivated should start digging into that and fix it but the goal of this PR was to not expose this to the user-facing API...